### DISCUSSION TITLE
Feedback about lists for starred repos

### DISCUSSION THREAD

**Discussion by Yunir** [community/community#9302] [Other Feature Feedback, Questions, & Ideas] [answered]
1. There is no possibility to sort, filter inside specific list. It's only possible to do for all of the starred repos.
2. In all starred repos page there is no possibility to show unlisted repos or repos for specific list only
3. Why there are separate pages for the lists? It would just good to filter by list name on the general 'all-starred-repos' page. This would help to solve the issue 1 too. It would be good to introduce something like "list:name" or via dropdown menu.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note @Yunir! I've passed these suggestions along to the team to help us prioritize what we build next.
